### PR TITLE
Created config server

### PR DESCRIPTION
**Pull request by saphal4859**
1.To fetch the props (testing purpose) : http://localhost:8081/fetchprops
2.To fetch props from config server : http://localhost:8071/product-service/dev
3.segregated service wise config files , common properties for all ms , common properties of all envs of a specific microservice

### CODE REVIEW

File: product-app/pom.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+  <artifactId>product-app</artifactId>
+
+  <properties>
+    <maven.compiler.source>17</maven.compiler.source>
+    <maven.compiler.target>17</maven.compiler.target>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+  </properties>
+
+</project>
```
Comment (yogi3345) [inline product-app/pom.xml:20]: Why another POM? why is here along with src folder?

**Comment by yogi3345** [comment]
To merge this we first need a config server. We might have to create a repo for it first.

**Comment by saphal4859** [comment]
> To merge this we first need a config server. We might have to create a repo for it first.

Yes , I have done changes for config server too , please create a repo for it and let me know

File: product-app/pom.xml (hunk 1 above)
Comment (saphal4859) [reply product-app/pom.xml:20]: Its the module pom

**Comment by yogi3345** [comment]
Let's see a demo, then we can merge

**Pull request merged**